examples/nim.cfg

```diff
@@ -1 +1 @@
--d:"wxWidgetsPath:../../wxWidgets-3.0.2"
+--d:"wxWidgetsPath:../../wxWidgets-3.0.2"
```

Comment:
Why the double `--`? It should be either one of these:
* `--define`
* `-d`

As I understand, the syntax is **exactly** as if written on the command line